colp_build/01_dataloading.sh

```diff
@@ -1,13 +1,9 @@
 #!/bin/bash
 source config.sh
 
-docker run --rm\
-    -v `pwd`:/home/colp_build\
-    -w /home/colp_build\
-    -u $(id -u ${USER}):$(id -g ${USER}) \
-    -e RECIPE_ENGINE=$RECIPE_ENGINE\
-    -e BUILD_ENGINE=$BUILD_ENGINE\
-    -e EDM_DATA=$EDM_DATA\
-    nycplanning/cook:latest bash -c "python3 python/dataloading.py"
+import_public dcp_pluto &
```

Comment:
do we have to fix pluto version? based on the python code we were loading 20v6. if so:
`import_public dcp_pluto 20v6 &`

Reply:
We used to take it from EDM_DATA, and I don't think there was a "latest" view.

Reply:
ahh I see